tools/flake8_plugin/setup.py

```diff
@@ -0,0 +1,36 @@
+from __future__ import with_statement
+import setuptools
+
+requires = [
+    'flake8 > 3.0.0',
+]
+
+flake8_entry_point = 'flake8.extension'
+
+setuptools.setup(
+    name='flake8-igz',
+    license='MIT',
+    version='0.1.0',
+    description='iguazio\'s twisted extension to flake8',
+    author='Omri Harel',
+    author_email='omrih@iguaz.io',
```

Comment:
Change to your credentials here.
No offense to Omri meant, but we're maintaining this now and the email of course is no longer relevant